File: Indixpert-Sept-batch2025/C Languege/function/with return-argument/task10.c
#include<stdio.h>
int charcount(char,char[]);
int main()
{
    char data[50],ch;
    int result;

    printf("Enter Your String = ");
    gets(data);

    printf("Enter Character = ");
    scanf("%c",&ch);

    result = charcount(ch,data);

    printf("%c = %d",ch ,result);

    return 0;
}
int charcount (char ch, char data[])
{
    int i,count=0;

    for (i = 0; data[i] != '\0'; i++)
    {
        if (data[i] == ch)
        {
            count++;
        }
            
    }

    return count;
}

File: Indixpert-Sept-batch2025/Python/nestes-if-else/task10.py
unit = int(input("Enter Your Value ="))

if (unit >= 50):
    res=unit*0.50
   
elif (unit >= 100):
    res=unit*0.75
   
elif (unit >= 200):
    res=unit*1.20
   
else:   
    res=unit*1.50
sc=res*0.2

print(sc+res)

